src/test/java/com/icthh/xm/ms/balance/web/rest/BalanceResourceIntTest.java

```diff
@@ -139,8 +139,7 @@ public void BeforeTransaction() {
     }
 
     @After
-    @Override
-    public void finalize() {
+    public void tearDown() {
```

Comment:
why name of the method is changed?

Reply:
method `finalize` is deprecated since `java 9`. Don't know need to use it in the tests. General practise is to name `setup/tearDown` methods for test inialization and destroy functionality